src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/ProvideCorrectArgumentsToFormattingMethodsTests.cs

```diff
@@ -63,82 +81,181 @@ public class C
 {
     void Method()
     {
-        Console.Write("""", 1);
-        Console.Write(""{0}"", 1, 2);
-        Console.Write(""{0} {1}"", 1, 2, 3);
-        Console.Write(""{0} {1} {2}"", 1, 2, 3, 4);
-        Console.Write(""{0} {1} {2} {3}"", 1, 2, 3, 4, 5);
+        {|#0:" + invocation + @"(""{1}"", 1, 2, 3)|};
```

Comment:
@jmarolf here I need to report on all extra args + somewhere for the missing format indexes (any suggestion?)

Reply:
for this case I think the most useful location would be `1,` and `3` indicating that these are unused.

---
In reply to: [508701363](https://github.com/dotnet/roslyn-analyzers/pull/4226#discussion_r508701363) [](ancestors = 508701363)